js/app.js

```diff
@@ -0,0 +1,34 @@
+$(document).on("click", "div[ng-click]", function() {
+
+  if($(this).attr("ng-click") == "underground = 'true'") locache.set("state", ["underground = 'true'"]);
```

Comment:
I would probably use [data attributes](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes) for the conditional.